test/functional/api/v2/hostgroups_controller_test.rb

```diff
@@ -18,13 +25,43 @@ class Api::V2::HostgroupsControllerTest < ActionController::TestCase
     assert !show_response.empty?
   end
 
+  def last_hostgroup
```

Comment:
Shall I also move other helpers like `basic_attrs` and `valid_attrs`? It really does not make any sense to me.